pymeasure/instruments/etslindgren/emcenter.py

```diff
@@ -0,0 +1,53 @@
+#
+# This file is part of the PyMeasure package.
+#
+# Copyright (c) 2013-2022 PyMeasure Developers
+
+#
+# Permission is hereby granted, free of charge, to any person obtaining a copy
+# of this software and associated documentation files (the "Software"), to deal
+# in the Software without restriction, including without limitation the rights
+# to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+# copies of the Software, and to permit persons to whom the Software is
+# furnished to do so, subject to the following conditions:
+#
+# The above copyright notice and this permission notice shall be included in
+# all copies or substantial portions of the Software.
+#
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+# IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+# FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+# AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+# LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+# OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+# THE SOFTWARE.
+#
+
+
+import logging
+
+from pymeasure.instruments import Instrument
+
+
+log = logging.getLogger(__name__)
+log.addHandler(logging.NullHandler())
+
+
+class EMCenter(Instrument):
+    """Represents ETS-Lindgren EMCenter main frame.
+    used for direct access to the instrument, does not control sub-modules
+    """
+
+    local = Instrument.setting(
```

Comment:
This property is a "setting" one, but it just ignores what is passed. Is this intentional ?
Normally you need to use this as follow:
`inst.local = some_value`
The command "LOCAL" will be issued regardless of the value of some_value.

Reply:
Agreed, assuming there is a "mode" other than "local", this should be a setting/control called "mode" (which then accepts values "local" or "something else". 
If there is only local mode, this should be a method (e.g. `set_local_mode(self)`)